docs/instrumentation-list.yaml

```diff
@@ -286,6 +390,11 @@ elasticsearch:
       javaagent:
       - org.elasticsearch:elasticsearch:[6.0.0,8.0.0)
       - org.elasticsearch.client:transport:[6.0.0,)
+  - name: elasticsearch-rest-common-5.0
+    srcPath: instrumentation/elasticsearch/elasticsearch-rest-common-5.0
```

Comment:
we will need to come up with logic to exclude these common modules. I will follow up on that.